pkgs/development/python-modules/pyrdfa3/default.nix

```diff
@@ -1,51 +1,48 @@
 { lib
 , buildPythonPackage
 , fetchPypi
-, fetchpatch
-, isPy27
-, rdflib
 , html5lib
+, pythonOlder
+, rdflib
+, setuptools
 }:
 
 buildPythonPackage rec {
   pname = "pyrdfa3";
   version = "3.5.3";
-  format = "setuptools";
-  disabled = isPy27;
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
 
   src = fetchPypi {
     inherit version;
     pname = "pyRdfa3";
     hash = "sha256-FXZjqSuH3zRbb2m94jXf9feXiRYI4S/h5PqNrWhxMa4=";
   };
 
-  patches = [
-    (fetchpatch {
-      name = "CVE-2022-4396.patch";
-      url = "https://github.com/RDFLib/pyrdfa3/commit/ffd1d62dd50d5f4190013b39cedcdfbd81f3ce3e.patch";
-      hash = "sha256-prRrOwylYcEqKLr/8LIpyJ5Yyt+6+HTUqH5sQXU8tqc=";
-    })
-  ];
-
```

Comment:
Why remove the CVE patch?

Reply:
Reverted. The patch should stay as there was no new release published which contain the change.

Thanks for bringing this up..